website/content/commands/index.mdx

```diff
@@ -254,3 +254,19 @@ products that support Enterprise namespaces.
 ```
 CONSUL_NAMESPACE=default
 ```
+
+## Accepts URL-Invalid Characters
```

Comment:
Does this section seem to be in the right place on this page? Or should it be higher/lower?

Reply:
I'm all for us putting it before the environment variables.  That way we transition from CLI behaviors to environment variables vs jumping back into CLI behavior context after reveiwing env vars.